tests/SprykerEcoTest/Zed/Payone/_support/_generated/PayoneZedTesterActions.php

```diff
@@ -1,4 +1,4 @@
-<?php  //[STAMP] 6d265cc3529a8d3f945309a25b014809
```

Comment:
Shouldn't this be ignored by git?

Reply:
have no idea, i can add to .gitignore if you request ?